process/mock/finalityAttesterStub.go

```diff
@@ -0,0 +1,25 @@
+package mock
+
+import (
+	"github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+)
+
+type FinalityAttesterStub struct {
+	GetFinalHeaderCalled func(shardID uint32) (data.HeaderHandler, []byte, error)
+}
+
+func (fas *FinalityAttesterStub) GetFinalHeader(shardID uint32) (data.HeaderHandler, []byte, error) {
+	if fas.GetFinalHeaderCalled != nil {
+		return fas.GetFinalHeaderCalled(shardID)
+	}
+
+	return &block.Header{
```

Comment:
define header separately like in nilFinalityAttester

Reply:
done